src/assets/data/programs.json

```diff
@@ -25,4 +25,10 @@
         "url": "https://www.dotnetconf.net/",
         "end_date": "2022-11-10"
     }
```

Comment:
```suggestion
    },
```

Reply:
In this case no, this causes a JSON PARSE error